spec/datadog/core/telemetry/collector_spec.rb

```diff
@@ -10,6 +10,7 @@
 require 'datadog/core/telemetry/v1/integration'
 require 'datadog/core/telemetry/v1/product'
 require 'ddtrace/transport/ext'
+require 'datadog/profiling/profiler'
```

Comment:
Minor: Is this really needed? I don't see any references to profiling in the collector, other than to its settings which are not under `datadog/profiling/profiler`.

Reply:
When running `bundle exec rake spec:main` was complaining that the constant was missing.

Let me double check

Reply:
Aaahh you're right. I had only looked at the `environment_logger.rb` and tried to run the spec and did not see the same result you did.

I missed that:
1) The `environment_logger_spec.rb` directly mentions the profiler
2) The `Datadog::Profiling::Profiler` class is only loaded [if profiling is supported](https://github.com/DataDog/dd-trace-rb/blob/7632a804d7a31ea706ac3b42fa46a449b67b8962/lib/datadog/profiling.rb#L191). This is why the spec worked for me but failed for you -- on my machine the profiler is set up correctly so it's supported, and on yours it hasn't been compiled :)